bin/packages/get-babel-config.js

```diff
@@ -11,13 +11,6 @@ const { options: babelDefaultConfig } = babel.loadPartialConfig( {
 	configFile: '@wordpress/babel-preset-default',
 } );
 const plugins = babelDefaultConfig.plugins;
-if ( ! process.env.SKIP_JSX_PRAGMA_TRANSFORM ) {
-	plugins.push( [ '@wordpress/babel-plugin-import-jsx-pragma', {
-		scopeVariable: 'createElement',
-		source: '@wordpress/element',
-		isDefault: false,
-	} ] );
-}
```

Comment:
With this present I got an `Error: Duplicate plugin/preset detected.` when trying to execute a build (`npm run start`). Removing this line fixed the build. It's not clear to me why this is essentially a duplicate of what is included in `@wordpress/babel-preset-default` but it _may_ be related to the fact that `wp-scripts` got an update in this pull which may in turn pull in a more recent version of `@wordpress/babel-preset-default`.

Reply:
Seems likely. `wp-scripts` went from 3.4.0 to 12.0.0. I'm not sure how Renovate let that happen.